streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBStore.java

```diff
@@ -312,6 +318,11 @@ public synchronized void put(final Bytes key,
     public void putAll(final List<KeyValue<Bytes, byte[]>> entries) {
         try (final WriteBatch batch = new WriteBatch()) {
             dbAccessor.prepareBatch(entries, batch);
+            // FIXME Will the recordMetadata be the offset of the last record in the batch?
```

Comment:
@vvcephei  What the comment is asking basically: What is the recordMetadata when there is a batch of records?